decidim-initiatives/app/commands/decidim/initiatives/create_initiative.rb

```diff
@@ -68,21 +60,14 @@ def build_initiative
           title: { current_locale => form.title },
           description: { current_locale => form.description },
           author: current_user,
-          decidim_user_group_id: form.decidim_user_group_id,
           scoped_type:,
-          area:,
-          signature_type: form.signature_type,
-          signature_end_date:,
-          state: "created",
-          hashtag: form.hashtag
+          signature_type: form.type.signature_type,
+          state: "created"
         )
       end
 
       def scoped_type
-        InitiativesTypeScope.find_by(
-          type: form.initiative_type,
-          scope: form.scope
-        )
+        InitiativesTypeScope.find_by(type: form.type)
```

Comment:
I understand that the scope has been removed but this may lead to random order of the `InitiativesTypeScope` due to the PostgreSQL default order changing when the database is updated. In case that table is updated, PostgreSQL can sometimes reorder the items into a seemingly random order (depending on how the records are persisted onto the storage device). See #7432 and #7424 for more information on this issue.

So if the idea here is to always get the first item, I would add an explicit order in front of the query:
```suggestion
        InitiativesTypeScope.order(:id).find_by(type: form.type)
```